tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -934,7 +934,7 @@ class MklFusedBatchNormOp : public OpKernel {
   bool is_training_;
   T* mean_values_;
   T* variance_values_;
-  size_t depth_;  // batch normalization is done for per channel.
+  int depth_;  // batch normalization is done for per channel.
 
```

Comment:
This is not the only time that size_t is being used in the MKL kernels. Did you change it here because it was throwing an error?

Reply:
@gzmkl can you verify this?

Reply:
Any updates here @gzmkl ?